### PR TITLE
Design Review - style(App): Fixes having the unnecessary whitespace below the FooterCfGov

### PR DESCRIPTION
**Pull request by shindigira**
closes https://github.com/cfpb/sbl-frontend/issues/365

## Changes
- style(App): Fixes having the unnecessary whitespace below the FooterCfGov

## How to test this PR

1. `yarn install` to get the latest DSR update
2. Go to any page where FooterCfGov is used (basically all)
3. Use mobile view and swipe up at the scroll end OR zoom out (CMD + - multiple times)
4. See how the footer's grey part still extends to the bottom

## Screenshots BEFORE the fix
Screenshot 2024-04-02 at 12 29 43 PM
Screenshot 2024-04-02 at 12 31 41 PM

## Screenshot - Fix B (View zoomed out) - Maximizing the footer 
Screenshot 2024-04-02 at 10 15 16 PM

### CODE REVIEW

**Review by billhimmelsbach — CHANGES_REQUESTED** [review]
Heyo! I think putting the background on a div restricted to the size of this div with `h-dvh` has some weird effects on pages that are larger than the viewport:
Screenshot 2024-04-02 at 6 27 47 PM

Digging through git, it looks like this `h-dvh` was set to help with making loading screens take up the viewport, but if you put this on a div that was set to `auto`, this approach works. 🎉

**Comment by natalia-fitzgerald** [comment]
@shindigira 
I prefer Fix B - extending the gray background - Maximizing the footer.

File: src/components/FooterCfGovWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { FooterCfGov } from 'design-system-react';
+
+function FooterCfGovWrapper(): JSX.Element {
+  // TODO: Choose between:
+  // Maximizing the midsection white space with 'mt-auto' or maximizing the gray space in the footer with 'flex-1
+  return <FooterCfGov className='flex-1' />;
```
[the remark this answers is not in the document]
Comment (natalia-fitzgerald) [reply src/components/FooterCfGovWrapper.tsx:6]: @shindigira - How can I see it?

[the remark this answers is not in the document]
Comment (billhimmelsbach) [reply src/components/FooterCfGovWrapper.tsx:6]: This is up for your review on the preview site @natalia-fitzgerald 👍

**Review by natalia-fitzgerald — APPROVED** [review]
Thanks for addressing this. Looks good @shindigira!

**Review by billhimmelsbach — APPROVED** [review]
Looks good! 🎉

**Pull request merged**